SocketRocket/SRWebSocket.m

```diff
@@ -597,18 +650,32 @@ - (void)send:(nullable id)message
 
 - (BOOL)sendString:(NSString *)string error:(NSError **)error
 {
+    return [self sendString:string error:error completion:nil];
+}
+
+- (BOOL)sendString:(NSString *)string completion:(nullable SRSendCompletionBlock)completion {
+    return [self sendString:string error:NULL completion:completion];
+}
+
+- (BOOL)sendString:(NSString *)string error:(NSError **)error completion:(nullable SRSendCompletionBlock)completion {
```

Comment:
What do you think if we would remove `error` argument from this method and move it only to the method that needs it? Thus we can eliminate the need for weird method with synchronous and asynchronous signatures?